### DISCUSSION TITLE
Google Drive : This app is blocked

### DISCUSSION THREAD

**Discussion by markstothard** [iterate-ch/cyberduck#16203] [Q&A]
I'm trying to use Mountain Duck with Google Drive, however, I receive this error message

"This app is blocked
This app tried to access sensitive info in your Google Account. To keep your account safe, Google blocked this access."

I've tried to apps Cyberduck or Mountain Duck within Manage the Marketplace app allowlist for your organization

but these apps are not listed.

Any suggestions, thank you

**Comment by dkocher** [comment]
We are working on restoring access to Google Drive and Google Storage. Duplicate for #16178.

**Reply by markstothard** [reply]
thank you for this update. I've followed #16178
